### DISCUSSION TITLE
How to merge patches stack from another branch?

### DISCUSSION THREAD

**Discussion by TD-Sky** [stacked-git/stgit#359] [Q&A] [answered]
Is there only one way (`stg export` and then `stg import`) to do that?

**Comment by jpgrayson** [comment, marked as answer]
`stg pick` and `stg sync` both deal with cross-branch patch management. Do either of those work for your use case?

**Reply by TD-Sky** [reply]
Sorry for my late reply because I'm so busy that I forgot you have answered.

These commands are amazing! Maybe I'm relying too much on the example commands in the documentation. I should RTFM.
